src/main/java/com/google/sps/data/DatastoreScheduledInterviewDao.java

```diff
@@ -123,4 +122,15 @@ public Entity scheduledInterviewToEntity(ScheduledInterview scheduledInterview)
     scheduledInterviewEntity.setProperty("interviewee", scheduledInterview.intervieweeEmail());
     return scheduledInterviewEntity;
   }
+
+  /** Updates a scheduledInterview Entity from a scheduledInterview object. */
+  public Entity scheduledInterviewToUpdateEntity(ScheduledInterview scheduledInterview) {
```

Comment:
I don't think the description here is accurate since this does not actually update anything, just creates a new entity.

Related to that, I think `scheduledInterviewToEntity` or `scheduledInterviewToDatastoreEntity` would be a more appropriate name.

Reply:
It's the method used in to update a stored scheduledInterview entity without losing to generated id. scheduledInterviewToEntity is already a method that creates a scheduledInterview entity with a generated id

Reply:
In that case, please rename to `scheduledInterviewToEntityForUpdate` or `scheduledInterviewToEntityWithId`. `scheduledInterviewToUpdateEntity` sounds too much like an "UpdateEntity" is a special kind of entity, which is not really accurate.